zulipterminal/ui_tools/boxes.py

```diff
@@ -1926,7 +1930,10 @@ def __init__(
             urwid.Text([" ", INVALID_MARKER, " No Results"]), "search_error"
         )
         urwid.connect_signal(self, "change", update_function)
-        super().__init__(caption=self.search_text, edit_text="")
+        if not fixed_search_caption:
+            super().__init__(caption="", edit_text=self.search_text)
```

Comment:
Ahh... This was changed recently when we decided to remove the search hint when the user presses `p` again. If we go by the current approach then I don't think we need this commit(?), but if we want to show the search caption when the user presses `p` too, then this will be useful.

Reply:
I am still convinced that we dont need to show the search hint when the user presses `p`, so I've dropped this commit and moved the "initial focus in the search-box" part to the emoji picker commit earlier.